### ISSUE TITLE
whitespace minification on application source code breaks `({query }) => ...`

### ISSUE THREAD

**Issue by paperdave** [elysiajs/elysia#724]
### What steps can reproduce the bug?

```
// @bun
import Elysia from "elysia";
var app = new Elysia;

//             | space missing here breaks
app.get("/", ({query }) => { console.log("Query:", query) });

app.listen(4122);
```

### What is the expected behavior?

receive query parameters

### What do you see instead?

`undefined`

### Additional information

Originally reported here https://github.com/oven-sh/bun/issues/12594

**Comment by SaltyAom** [comment]
Should have been fixed with 69eb72a, published under 1.1.2

**Comment by mavyfaby** [comment]
I can confirm this works now:
![image](https://github.com/user-attachments/assets/8e249ac2-15f6-42a4-9f6b-4c299a19e1d2)

Thanks @SaltyAom
